src/components/pages/devices/flyouts/deviceNew/deviceNew.js

```diff
@@ -292,11 +292,23 @@ export class DeviceNew extends LinkedComponent {
   }
 
   deviceTypeChange = ({ target: { value } }) => {
-    this.props.logEvent(toSinglePropertyDiagnosticsModel('Devices_DeviceTypeSelect', 'DeviceType',
+    this.props.logEvent(toSinglePropertyDiagnosticsModel('Devices_DeviceType_Select', 'DeviceType',
```

Comment:
couple more instances of name changes to be reverted: DeviceTypeSelect, ApplyClick, TopXCloseClick,, CancelClick, CloseClick